### DISCUSSION TITLE
Howbto install rust cargo and all that

### DISCUSSION THREAD

**Discussion by Greenarrow4245** [nix-community/nix-on-droid#303] [Q&A]

**Comment by t184256** [comment]
nix-on-droid assumes some Nix familiarity.

I'd start with either adding `rustc` and `cargo` to your package list or just requesting a shell with them using `nix shell -p cargo -p rustc` (non-flakes) / `nix shell nixpkgs#cargo nixpkgs#rustc' (flakes).

Then I'd learn how Rust is handled the Nix way: https://nixos.org/manual/nixpkgs/stable/#rust

Hope that helps.
